internal/pikamon/spawn/pokemon.go

```diff
@@ -17,18 +17,10 @@ type pokemonSpawner struct {
 	debounceWindow time.Duration
 }
 
-func newPokemonSpawner(chance float64, debounceWindow time.Duration) (*pokemonSpawner, error) {
-	cache, err := ristretto.NewCache(&ristretto.Config{
-		NumCounters: 1e2,
-		MaxCost:     1e2,
-		BufferItems: 64,
-	})
-	if err != nil {
-		return nil, errors.Wrap(err, "failed to create channel cache")
-	}
+func newPokemonSpawner(botCache *util.BotCache, chance float64, debounceWindow time.Duration) (*pokemonSpawner, error) {
 	return &pokemonSpawner{
 		chance:         chance,
-		channelCache:   cache,
+		channelCache:   botCache.ChannelCache,
```

Comment:
`botCache.ChannelCache` kind of stutters, I'd suggest making it `botCache.Channels`.

Reply:
I will tackle this as part of #24 since I will likely have to adjust this anyways